### PR TITLE
feat(cypress): support cypress v8

### PR DESCRIPTION
**Pull request by kentcdodds**
**What**: Upgrades to cypress v8

**Why**: To stay on the latest

**How**: just a quick upgrade in the package.json

**Checklist**:

- [ ] Documentation N/A
- [ ] Tests N/A
- [x] Ready to be merged

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -15,11 +15,11 @@ export interface CTLMatcherOptions
...
 }
 
-export type MatcherOptions = DTLMatcherOptions | CTLMatcherOptions
-export type ByRoleOptions = DTLByRoleOptions | CTLMatcherOptions
+export type MatcherOptions = CTLMatcherOptions | DTLMatcherOptions
+export type ByRoleOptions = CTLMatcherOptions | DTLByRoleOptions
 export type SelectorMatcherOptions =
-  | DTLSelectorMatcherOptions
   | CTLMatcherOptions
+  | DTLSelectorMatcherOptions
```
Comment (kentcdodds) [inline types/index.d.ts:22]: This change was unnecessary really. I enabled a stupid lint rule as a warning that I should probably disable... 

Just did... I'll revert this bit...

**Comment by nickmccurdy** [comment]
Thanks but this was already fixed in #196.

**Comment by kentcdodds** [comment]
lol, my bad 😅

**Comment by nickmccurdy** [comment]
It's ok, there were 3 PRs with the same code somehow, it's not just you :P

**Pull request closed without merging**